packaging/preupgrade-assistant.spec

```diff
@@ -341,9 +349,10 @@ fi
 %endif # build_ui
 
 %if 0%{?rhel} && 0%{?rhel} <= 5
-%{python_sitelib}/preupg_pykickstart/
+%{python_sitelib}/%{bundled_pykickstart_dirname}/
 %{_bindir}/preupg_ksvalidator
 %{_bindir}/preupg_ksverdiff
+%license %{_docdir}/%{bundled_pykickstart_dirname}/COPYING
```

Comment:
missing owner of
%{_docdir}/%{bundled_pykickstart_dirname}

for fix, add line (before the one):
dir %{_docdir}/%{bundled_pykickstart_dirname}